libs/net/kafka/Models/ContentBase.cs

```diff
@@ -56,6 +56,16 @@ public abstract class ContentBase
     /// </summary>
     public string FilePath { get; set; } = "";
 
+    /// <summary>
+    /// get/set - The section containing the content.
+    /// </summary>
+    public string Section { get; set; } = "";
```

Comment:
Not ideal for a base class as they're really only used in print content.  But we'll leave for now.